src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -15163,6 +15166,34 @@ public IEnumerator GetEnumerator()
                 );
         }
 
+        [Fact]
+        public void CS1622ERR_ReturnInIterator_WrongType()
+        {
+            var text = @"
```

Comment:
nit: raw strings.

Reply:
If there's a need to make a functional change to the test I may go ahead and fix it up. But otherwise, as a copy/paste/modify job, I will leave as is.